lib/krikri/parser.rb

```diff
@@ -162,6 +187,18 @@ def field(*args)
       end
 
       ##
+      # Accesses the union of multiple specified fields.
+      #
+      # @return [ValueArray] an array containing the nodes available in the
+      # given fields.
+      def fields(*args)
```

Comment:
If I'm reading this and the spec correctly, something like `records.fields(:foo, :bar)` (i.e, without explicit array typing) should return the union for the fields `:foo` and `:bar`. Correct?